packages/mask/src/index.js

```diff
@@ -137,7 +137,7 @@ export function stripDown(template, input) {
         }
 
         for (let j = 0; j < inputToBeStripped.length; j++) {
-            if (inputToBeStripped[j] === template[i]) {
+            if (inputToBeStripped[j] === template[i] && template[i] !== inputToBeStripped[i]) {
```

Comment:
This can never evaluate to true.

It's saying "if these match and if they also don't match" which is impossible.

Reply:
Hi @ekwoka - Thanks for the quick feedback.

I disagree, note that the new conditional is not using the same iterator for inputToBeStripped (we are using i not j)

From `mask.spec.js:L11`
```js

    expect(stripDown('(999) 999-9999', '716 2256108')).toEqual('7162256108')

template = "(999) 999-9999"
input = "716 2256108"

// when i = 5 and j = 3 we see inputToBeStripped[j] = " " and template[i] = " " (so first condition match), 
// then we see inputToBeStripped[i] = 2, 

// This matches the condition the space is removed.

```

Reply:
Ah yes, you're right. That's hard to read.